README.md

```diff
@@ -415,8 +411,8 @@ This message is used to communicate the current sonar state. If the data field i
 | u8 | gain_setting | Analog gain setting (0 = low, 1 = normal, 2 = high) |  |
 | u16 | angle | Head angle | gradian |
 | u16 | transmit_duration | Acoustic transmission duration (1~1000 microseconds) | microsecond |
-| u16 | interval | Time interval between individual signal intensity samples in 25nsec increments (40 to 40000 == 1 microsecond to 1000 microseconds) |  |
-| u16 | transmit_frequency | Acoustic operating frequency |  |
+| u16 | interval | Time interval between individual signal intensity samples in 25nsec increments (80 to 40000 == 2 microseconds to 1000 microseconds) | nanosecond |
```

Comment:
wait, 80 nanoseconds is 2 microseconds ? what ? 
Using nanoseconds here as unit is really weird.

Reply:
I can remove the units or change it to `nanoseconds x 25`